src/gilded_rose.spec.js

```diff
@@ -155,4 +155,21 @@ describe('generating ordered lists of items', () => {
     const testOutput = getHtmlListFromArray(list);
     expect(testOutput).toBe(`<ol><li>${JSON.stringify(sulfuras)}</li></ol>`);
   });
+  it('updates items and generates an ordered list with the updated items', () => {
```

Comment:
One minor note on formatting: it looks like there are empty lines between tests throughout this file. It doesn't _really_ matter whether there are empty lines between each test or not but I'd try to make it consistent.